index.html

```diff
@@ -2646,8 +2684,9 @@ <h5>Conclusion</h5>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
       Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-  <!-- short summary of how the tag applies -->
+<li>
+<a data-ucr-role="#author-extensible"></a>
```

Comment:
```suggestion
<a data-ucr-role="#author-extensibility"></a>
```

(Maybe the id should be extensible, but it's currently extensibility!)

Reply:
I have made this mistake every time I tried to type this tag, made a mental note not to do it again, and proceeded to do it again 🤦‍♀